internal/node/node_repositories.bzl

```diff
@@ -20,9 +20,10 @@ See https://docs.bazel.build/versions/master/skylark/repository_rules.html
 
 load("//internal/common:check_bazel_version.bzl", "check_bazel_version")
 load("//internal/common:check_version.bzl", "check_version")
-load("//internal/common:os_name.bzl", "os_name")
+load("//internal/common:os_name.bzl", "OS_NAMES", "os_name")
 load("//internal/npm_install:npm_install.bzl", "yarn_install")
 load("//third_party/github.com/bazelbuild/bazel-skylib:lib/paths.bzl", "paths")
+load("//toolchains/node:node_configure.bzl", node_toolchain_configure = "node_configure")
```

Comment:
Seems like a better name for the .bzl file and repository_rule is `node_toolchain_configure` so it is less ambigious as to what it does?

Reply:
It is already within the toolchains directory so I thought that would be clear from there. But can still rename if you prefer. Let me know.

Reply:
I like the explicit naming so its clear that its configuring a `node_toolchain`. Will avoid the `node_toolchain_configure = "node_configure"` on `node_repositories.bzl`. I'll push this along with a few last touches.